src/apps/fuzzers/libfuzzer.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2021 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdint.h>
+#include <string.h>
+#include <stdlib.h>
+#include <stdio.h>
+
+#include "h3api.h"
+#include "utility.h"
+
+int
+LLVMFuzzerTestOneInput(const uint8_t *data, size_t size)
+{
+    char *new_str = (char *)malloc(size+1);
+    if (new_str == NULL){
+        return 0;
+    }
+    memcpy(new_str, data, size);
+    new_str[size] = '\0';
+
+    
+    H3Index h3;
+    h3 = H3_EXPORT(stringToH3)(new_str);
+    
+    H3Index input[] = {h3, h3};
+    int inputSize = sizeof(input) / sizeof(H3Index);
+
+    // fuzz compactCells
+    H3Index* compacted = calloc(inputSize, sizeof(H3Index));
+    int err = compactCells(input, compacted, inputSize);
```

Comment:
Not sure how rigorously this will be able to test `compactCells` since it always uses two inputs that are the same cell, but this seems like a good idea we can add other functions to.

Reply:
We can add several random `H3Index` in `input[]`.

For now, though, I think it should be good for an initial setup.

Reply:
Closing. Feel free to reopen.